config/app.ts

```diff
@@ -1,19 +1,39 @@
 import express from 'express';
+import morgan from 'morgan';
+
 import database from './database';
+import { logger, stream } from './logger';
 
 const app: express.Application = express();
 const port = process.env.PORT || 3000;
 
+process
+	.on('unhandledRejection', (reason, promise) => {
+		console.log('Unhandled Rejection at:', promise, 'reason:', reason);
+	})
+	.on('uncaughtException', (err) => console.log(`Caught exception: ${err}`));
+
+morgan.token('params', (req: express.Request): string => {
+	const propsArr = [];
+	for (const prop in req.params){
+		propsArr.push(prop);
+	}
+	return propsArr.join();
+});
+
+app.use(morgan(':params :method :url :response-time'));
+app.use(morgan('combined', { stream }));
 app.use(express.json());
 app.use('/users', require('../components/User/routers/UserRoute'));
 app.use('/groups', require('../components/Group/routers/GroupRoute'));
 
 app.listen(port, async () => {
 	try {
 		await database.authenticate();
-		await database.sync({force: true});
+		await database.sync({force: false});
+		await logger.info(`Listenning on port ${port}`);
+		
 	} catch (err) {
-		console.log(`Unable to connect to ${port}`);
+		logger.info(`Unable to connect to ${port}`);
```

Comment:
Actually you don't know exactly the place where the error has happened (maybe while DB connection?). So, your message could be misleading. Let's change the message to something specific, for example, you can simply log the error. Also, the logger level should be an `error` (at least). Maybe even `fatal`, because it doesn't make sense to continue running application without DB.